warehouse/templates/packaging/detail.html

```diff
@@ -78,14 +78,18 @@ <h3 class="package-snippet__title">
     {% set index_url = " -i https://test.pypi.org/simple/" %}
   {%- endif -%}
 
+  {%- if release.is_draft -%}
+    {% set extra_index_url = " --extra-index-url=https://pypi.org/draft/" + release.draft_hash + "/" %}
+  {%- endif -%}
+
   {%- if request_route == "packaging.release" -%}
     {% set project_version = "==" + release.version %}
   {%- endif -%}
 
   {%- if release.version.epoch -%}
-    pip install{{ index_url }} '{{ release.project.name }}{{ project_version }}'
+    pip install{{ index_url }} '{{ release.project.name }}{{ project_version }}' {{ extra_index_url }}
   {%- else -%}
-    pip install{{ index_url }} {{ release.project.name }}{{ project_version }}
+    pip install{{ index_url }} {{ release.project.name }}{{ project_version }} {{ extra_index_url }}
```

Comment:
I just think `pip install my_package <some_very_long_option>` is easier to read than `pip install <some_very_long_option> my_package` when viewing this page (which is all this does)

Also, since `index_url` is only ever used for test.pypi.org, which may be deprecated with the addition of this feature~
So I don't expect to show these two together :)